druid-shell/src/web/mod.rs

```diff
@@ -166,7 +167,7 @@ fn setup_web_callbacks(window_state: &Rc<WindowState>) {
 
             let event = MouseEvent {
                 x: event.offset_x() as i32,
-                y: event.offset_x() as i32,
```

Comment:
.... figured out why half of the calc example buttons iddn't work... :roll_eyes: